### DISCUSSION TITLE
Handler Dependent(....) skipped

### DISCUSSION THREAD

**Discussion by LeSnow-Ye** [nonebot/discussions#326] [问答]
第三方插件无法使用
开 LOG_LEVEL=DEBUG 显示  Handler Dependent(....) skipped
![图片](https://user-images.githubusercontent.com/40481418/220881316-70c9abcf-a17c-4a45-ac69-039ddc587010.png)

**Comment by yanyongyu** [comment]
事件类型不对，你可以看插件能接收的事件类型
